src/builder.ts

```diff
@@ -25,6 +28,44 @@ const getWalletNonce = async (app: Application, config: Config): Promise<number>
     return walletNonce;
 };
 
+export const makeMultisignatureRegistration = async (config: Config): Promise<Contracts.Crypto.Transaction> => {
+    const { cli } = config;
+    const { multiSignatureRegistration, senderPassphrase } = cli;
+
+    const app = await getApplication(config);
+
+    const walletNonce = await getWalletNonce(app, config);
+
+    const { publicKeyFactory } = makeIdentityFactories(app);
+
+    const multisignatureAsset = {
+        min: multiSignatureRegistration.min,
+        publicKeys: await Promise.all(
+            multiSignatureRegistration.participants.map(
+                async (participant) => await publicKeyFactory.fromMnemonic(participant),
+            ),
+        ),
+    };
+
+    const transaction = await app
+        .resolve(MultiSignatureBuilder)
+        .fee(multiSignatureRegistration.fee)
+        .nonce((walletNonce + 1).toFixed(0))
+        .recipientId(await publicKeyFactory.fromMultiSignatureAsset(multisignatureAsset))
```

Comment:
The recipient should be an address, but actually isn't even needed for the registration so we can remove it.

Reply:
```suggestion
```